Remove CodePen link in Intermediate CSS - CSS Functions (#23550)
TheOdinProject/curriculum: html_css/intermediate_css/css_math_functions.md

`calc()` is handling the math for us even though we are mixing vh, rem and px units.
Combined with CSS variables, `calc()` can save us from the headache of repeating CSS rules.

> I encourage you to take a moment and edit the CodePen above to understand how `calc()` is being used. Play around with the different units and sizes of the elements and see what results you get before moving on.

Reviewer: Wording suggestion:

```suggestion
You should be able to grasp how `calc()` is used in the above CodePen embed. We encourage you to play around with different units and sizes of the elements to see what results you get before moving on.
```
